lib/pure/quitprocs.nim

```diff
@@ -1,28 +0,0 @@
-#
-#
-#            Nim's Runtime Library
-#        (c) Copyright 2016 Andreas Rumpf
```

Comment:
Why does this have a 2016 when it's a new module? (I assume added after 1.0?)

Reply:
pre-existing from before this PR but happy to change this to 2020.
(IIUC araq wrote quitprocs.nim in some private branch but it was never merged in nim until ~1 month ago)

Reply:
That is correct, it's old code I forgot to add.

Reply:
fixed